### PR TITLE
Toggle schemas and tables. FE, BE, the whole package.

### PR DESCRIPTION
**Pull request by anthony2261**
Initial Notes:
- what do we want the behavior to be when the DSN changes during a connection update? I say we enable everything we find in the new dsn, it could be a different database!
- With this change, the schema will always be shown, even for databases with one schema. For example, sqlite dbs can only have one schema named "main", and it will show up in the generated SQL, for example: `SELECT * from main.purchases`.
- Tested this with sqlite, postgres, sql server and mysql

https://github.com/user-attachments/assets/199ac3fd-fe4b-4495-b017-39b24ff207b5

### CODE REVIEW

**Review by RamiAwar — APPROVED** [review]

File: backend/dataline/services/llm_flow/utils.py [hunk 1]
```diff
@@ -1,59 +1,113 @@
...
-        schema: str | None = None,
+        schemas: list[str] | None = None,
         metadata: MetaData | None = None,
         ignore_tables: list[str] | None = None,
         include_tables: list[str] | None = None,
         sample_rows_in_table_info: int = 3,
         indexes_in_table_info: bool = False,
         custom_table_info: dict | None = None,
-        view_support: bool = True,  # default False in base class
+        view_support: bool = False,
```
[the remark this answers is not in the document]
Comment (anthony2261) [reply backend/dataline/services/llm_flow/utils.py:29]: This was just copied/pasted from that other github issue in langchain. Will set it back to True

[the remark this answers is not in the document]
Comment (anthony2261) [reply backend/dataline/services/llm_flow/utils.py:29]: Works like a charm! :D

image

File: backend/dataline/models/connection/model.py [hunk 2]
```diff
@@ -17,6 +17,7 @@ class ConnectionModel(DBModel, UUIDMixin, kw_only=True):
     type: Mapped[str] = mapped_column("type", String, nullable=False)
     dialect: Mapped[str | None] = mapped_column("dialect", String)
     is_sample: Mapped[bool] = mapped_column("is_sample", Boolean, nullable=False, default=False, server_default="false")
+    options: Mapped[dict[str, Any] | None] = mapped_column("options", JSON, nullable=True)  # type: ignore[misc]
```
[the remark this answers is not in the document]
Comment (anthony2261) [reply backend/dataline/models/connection/model.py:20]: Just copied it from `backend/dataline/models/message/model.py` 😋will make the change

[the remark this answers is not in the document]
Comment (anthony2261) [reply backend/dataline/models/connection/model.py]: I made the change, but it doesn't feel necessary. Check it out.

**Review by RamiAwar — APPROVED** [review]

**Pull request merged**